src/main/java/org/galatea/starter/entrypoint/IexRestController.java

```diff
@@ -6,8 +6,10 @@
 import lombok.extern.slf4j.Slf4j;
 import net.sf.aspect4log.Log;
 import net.sf.aspect4log.Log.Level;
-import org.galatea.starter.domain.IexLastTradedPrice;
 import org.galatea.starter.domain.IexSymbol;
+import org.galatea.starter.domain.IexHistoricalPrice;
+import org.galatea.starter.domain.IexLastTradedPrice;
```

Comment:
I don't think you are using last traded price, so avoid imports you aren't using

Reply:
last traded price is used within getLastTradedPrice. It was in the code before I touched anything